### PR TITLE
fix laser turret sprite

### PR DESCRIPTION
**Pull request by Brick63**
fix related to #4081 makes the sprite match the hard-point

### CODE REVIEW

**Comment by Brick63** [comment]
it has the PR, just took me a minute to open it

**Comment by Nescio0** [comment]
Copyright file?

The proposed image looks nice and certainly fixes #4081. However, let's compare it with the heavy laser and electron turrets:

electron turret 2x
heavy laser turret 2x
laser turret 2x

As you can see the proposed laser turret has a small gap between the two halves; to me this suggest that the two cannons might rotate vertically independently from each other, which ought not to be the case. By contrast, the heavy laser and electron sprites have here some solid piece, emphasizing the two cannons are supported by a turret mount.
In short, could you raise or enlarge the piece between the two halves?

**Comment by Pointedstick** [comment]
> As you can see the proposed laser turret has a small gap between the two halves; to me this suggest that the two cannons might rotate vertically independently from each other, which ought not to be the case. By contrast, the heavy laser and electron sprites have here some solid piece, emphasizing the two cannons are supported by a turret mount.
> In short, could you raise or enlarge the piece between the two halves?


I don't think this is actually a problem.

@Brick63 Please update the copyright file in this PR (as well as the HiDPI PR) and then I think we can merge it!

**Comment by jafdy** [comment]
It looks to me like the laser turret is angled slightly differently to the heavy and electron turrets. The copyright file doesn't need updating as he only moved a bit off the turret not made a new piece (as stated by brick himself in the discord).

**Comment by Brick63** [comment]
The original had that exact same dip there, only it was all on one side. I didn't design this, only altered it to fit something else and don't feel like that is deserving changing the copyright.

**Comment by MCOfficer** [comment]
> It looks good, but I do agree with Amazinite: Why do we need all the turrets to be of the same "two barrels on either side" type design? Wouldn't it be easier to change the sprite instead?

looking at it from an engineer's perspective, there's no reason not to design it like this. There's no real upside of pairing guns vertically, the symmetry likely makes for easier construction, and turret mounts (or their parts) are probably standardized to some degree. If we want this turret to be the odd one out, i feel like we need some kind of solid reason.

**Comment by Amazinite** [comment]
At that point I think you’re getting too detailed. Does every weapon description need to justify its design to you? No, it just needs to look good! Just because it looks different doesn’t mean it needs to explain why it looks different. Does the proton turret explain why it’s the only non-AM turret in human space with one gun on it? No, it just is.

**Comment by Pointedstick** [comment]
So what are we doing with this? The discrepancy between the sprite and the beam is a real issue. I'm okay with this sprite change, but then we also need a new hardpoint sprite too.

The alternative of making the beam only depict a single brighter laser rather then two side-by-side lasers could work too. But we should do _something_.

**Comment by Brick63** [comment]
That's the this, the hard point looks the same as what this sprite does, so it's in no need of change

**Comment by Amazinite** [comment]
I still think that the hardpoint sprite and projectile should change and not the outfit sprite. It may be "more work" to change two things than to change one thing, but it's also a less noticeable change than changing the outfit sprite. Changing the projectile and hardpoint sprite will only go noticed when actually fighting against or with the laser turret, which isn't exactly all that common past a certain point, while changing the outfit image is highly noticeable given that you can see laser turrets in outfitters at any point.

**Comment by Pointedstick** [comment]
That could work too. My only very minor concern with that approach is that it might be a bit weird having just this one turret fire one bright beam from the center when all the other beam-type turrets fire two side-by-side beams. But we can certainly try it out and see if it works.

**Comment by Nescio0** [comment]
Actually I still don't see what's wrong with this.
